src/liblsquic/lsquic_handshake.c

```diff
@@ -933,7 +935,7 @@ lsquic_enc_session_gen_chlo (lsquic_enc_session_t *enc_session,
     unsigned char pub_key[32];
     size_t ua_len;
     uint32_t opts[1];  /* Only NSTP is supported for now */
-    unsigned n_opts, msg_len, n_tags, pad_size;
+    unsigned n_opts, msg_len, n_tags, pad_size = 0;
```

Comment:
Necessary because of compiler bug (well it looks like a bug to me) - the only time it's used is under the same condition as to when it's set.  But it is what it is and thus a good change that should remain.